build.rs

```diff
@@ -310,11 +338,11 @@ fn download_static_lib_binaries() {
 
   let filename = static_lib_path();
   if filename.exists() {
-    println!("static lib already exists {}", filename.display());
+    println!("Deleting old static lib {}", filename.display());
     println!("To re-download this file, it must be manually deleted.");
```

Comment:
I believe this line can be removed, as the file is being deleted.

```suggestion
```

I think, though, that it's not a good idea to just delete and re-download the file, as there may be cases where one may want to use a custom static lib separately compiled. This is achieved by placing the custom file in the expected path. If the file is deleted, this custom case is not covered anymore.

Reply:
I have removed that line and added a new commit that hopefully supports the described use case.